joomla/joomla-cms
# Creating categories on the fly with numbers

Pull Request for Issue #22216.

### Summary of Changes

Adds prefix to categories created on the fly to allow creating categories starting with numbers.

### Testing Instructions

1. Create a category and remember the ID (or use an existing category) e.g. "2"
2. Create a new article or edit one and try to create a new category with the name: "2nd workshop" (important: the ID of the category from 1. has to be the leading character) by typing in the name in the category select on the right side + confirm with [ENTER]
3. Save article

### Expected result

A new category is created and assigned to the article

### Actual result

The old category from 1 is assigned

### Documentation Changes Required

New `Categoryedit` field property added.
Categories created on the fly contain `#new#` prefix.
